bin/zip.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+const archiver = require('archiver');
+
+const isDirectory = (x) => fs.statSync(x).isDirectory();
+
+const [zipPath, ...files] = process.argv.slice(2);
+
+const date = new Date('2009-06-12');
```

Comment:
Can you comment this line and maybe the file please? I can just see us in a year being like wtf.

Reply:
Done: 11162946d3c310f12cf1275d8b4b0e6b6d0dc4ac